src/mumble/MainWindow.h

```diff
@@ -180,9 +181,9 @@ class MainWindow : public QMainWindow, public Ui::MainWindow {
 
 	Search::SearchDialog *m_searchDialog = nullptr;
 
-	MenuLabel *m_localVolumeLabel;
-	UserLocalVolumeSlider *m_userLocalVolumeSlider;
-	ListenerLocalVolumeSlider *m_listenerLocalVolumeSlider;
+	std::unique_ptr< MenuLabel, decltype(&Mumble::QtUtils::deleteQObject) > m_localVolumeLabel;
```

Comment:
Maybe it would make sense to add a typedef for these kind of unique_ptr inside QtUrils:
```cpp
template< typename T >
using qt_unique_ptr< T > = std::unique_ptr< T, decltype(...)>;
```

Reply:
Yea, I have come up with this exact thing, too. I also wanted to create a factory function for ``qt_unique_ptr`` but this does not work, because it considers ``return`` a copy operation or something...

Reply:
https://github.com/mumble-voip/mumble/commit/4d5c1c4eae36f3fdef7bd6aeafed3e0eacfc8e86